src/Features/Core/Portable/EmbeddedLanguages/StackFrame/StackFrameParser.ParseResult.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.CodeAnalysis.EmbeddedLanguages.Common;
+
+namespace Microsoft.CodeAnalysis.EmbeddedLanguages.StackFrame
+{
+    internal partial struct StackFrameParser
+    {
+        private struct ParseResult<T>
+        {
+            public readonly bool Success;
+            public readonly T? Value;
```

Comment:
can these have one of those MemberNotNullWhen annotations?  i suppose not since Value can still be null in the success case right?

Reply:
Correct. Success doesn't indicate Value is non-null, it just determines if parsing should stop